frontend/src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -132,7 +132,7 @@ class WorkloadDetails extends React.Component<WorkloadDetailsPageProps, Workload
 
     if (!serverConfig.kialiFeatureFlags.disabledFeatures || !serverConfig.kialiFeatureFlags.disabledFeatures.includes('logs-tab')) {
       const logTab = (
-        <Tab title="Logs" eventKey={2} key={'Logs'}>
+        <Tab title="Logs" eventKey={2} key={'Logs'} data-cy={"workload-details-logs-tab"}>
```

Comment:
Same here, "data-cy" -> "data-tab-type" or something more meaningful with the value (no need to be specific to cypress).